### ISSUE TITLE
Feature request: rename devices and group them?

### ISSUE THREAD

**Issue by xCite1986** [michbeck100/pimatic-echo#3]
i think in some cases its useful to rename devices to a other name as the device name within pimatic and group them by seperate id

```
"devices": [
  {
    "id": "switch",
    "class": "DummySwitch",
    "name": "Workroom Switch",
    "echo": {
      "name": "Light",
      "room": "Workroom"
    }
  }
]
```

so alexa could handle something like:
"alexa, turn [name] in the [room] on" 
or 
"alexa, turn all in the [room] on"
or 
"alexa, turn in the [room] the [name] on"

**Comment by michbeck100** [comment]
Setting the room is currently not possible. You have to name your devices more unique. But you could also name your device "[name] in the [room]".
I'm leaving this issue open, as the name override is a neat feature.

**Comment by michbeck100** [comment]
Implemented with version 0.0.4

**Comment by xCite1986** [comment]
great, thanks for your work :)
